### ISSUE TITLE
XLS export negative numbers not exported correctly

### ISSUE THREAD

**Issue by mkoczorowski** [clarketm/TableExport#97]
When I export a table which have negative numbers in cell all of those numbers are incorrect in xls file.

an example comparing the same table exported to csv vs xls

![image](https://user-images.githubusercontent.com/8155896/33232315-efb5ca60-d1fb-11e7-9227-fdfbec1b3802.png)

what can cause this ? can anyone help ?

**Comment by saidbakr** [comment]
Indeed, it is not negative numbers only. It extends to any dash `-` character containing string, such as MySQL datetime format or text like `ABM-1`. Also unicode charcaters such as Arabic. However, in CSV and Text export format everything works fine.

**Comment by mkoczorowski** [comment]
hi, thanks for the explanation. Do you know is there any workaround ?

**Comment by saidbakr** [comment]
@mkoczorowski Unfortunately, I do not know else using CSV. However, CSV need proper import to MS Excell, but it works smoothly with Libre Office Calc
